ext/RastersArchGDALExt/resample.jl

```diff
@@ -88,6 +88,10 @@ function resample(A::RasterStackOrArray;
     # resample with `warp`
     resampled = warp(A, flags; kw...)
 
+    if !isnothing(to)
```

Comment:
This must have been added accidentally, right?

Reply:
Haha yes maybe from when we were demoing that change